### PR TITLE
style blogpost page

### PR DESCRIPTION
**Pull request by rccsousa**
Why: 
- blogposts page was missing the correct styling;
How:
- added new components to encapsulate different parts of blog posts and new methods to manipulate `dangerouslySetInnerHTML` and ensure all headings are enforced to h5 to maintain format consistency as well as labeling chapters;

### CODE REVIEW

File: src/app/_components/Share/index.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  const shareTo = async () => {
+    await navigator.share({
+      title: 'cenas',
+      text: 'mais cenas',
+      url: window.location.href,
+    })
+  }
+
+  const [url, setUrl] = useState('')
```
Comment (davelange) [inline src/app/_components/Share/index.tsx:24]: Can't we use `window.location.href` directly, instead of setting url in a useEffect?

**Review by davelange — APPROVED** [review]
Nice work 💯

File: src/app/_components/Share/index.tsx (hunk 1 above)
Comment (davelange) [reply src/app/_components/Share/index.tsx:24]: nvm, it's because of nextjs shenanigans, of course

Comment (rccsousa) [reply src/app/_components/Share/index.tsx:24]: yep, classic nextjs shenanigans 😅 
for future refference: for this type of projects would astro (or similar) be leaner / less bloated code-wise than nextjs?

**Pull request merged**